src/plugins/intel_cpu/src/nodes/kernels/riscv64/jit_uni_eltwise_generic.cpp

```diff
@@ -0,0 +1,537 @@
+            }
+        };
+
+        for (size_t i = 0; i < jep.inputs_number; i++) {
+            ld(reg_offsets, reg_const_params, GET_OFF(src_offsets) + i * sizeof(size_t));
+            ld(src_gpr(i), reg_const_params, GET_OFF(src_ptr[0]) + i * sizeof(size_t));
+            init_ptrs_with_offsets(src_gpr(i));
+        }
+
+        ld(reg_offsets, reg_const_params, GET_OFF(dst_offsets));
+        ld(dst_gpr(), reg_const_params, GET_OFF(dst_ptr));
+        init_ptrs_with_offsets(dst_gpr());
+
+        ld(reg_work_amount, reg_const_params, GET_OFF(work_amount));
+    } else {
+        auto init_ptrs_with_offsets = [&](Reg reg, const std::vector<size_t>& offsets) {
+            for (int j = 0; j < offset_count; j++) {
+                if (jep_.dims[j] != 1 && offsets[j] != 0) {
+                    uni_li(reg_tmp_0, static_cast<int>(offsets[j]));
+                    ld(reg_tmp_1, reg_indexes, static_cast<int>(j * sizeof(size_t)));
+                    mul(reg_tmp_0, reg_tmp_0, reg_tmp_1);
+                    add(reg, reg, reg_tmp_0);
+                }
+            }
+        };
+
+        for (size_t i = 0; i < jep.inputs_number; i++) {
+            ld(src_gpr(i), reg_const_params, GET_OFF(src_ptr[0]) + i * sizeof(size_t));
+            init_ptrs_with_offsets(src_gpr(i), jep.src_offsets[i]);
+        }
+
+        ld(dst_gpr(), reg_const_params, GET_OFF(dst_ptr));
+        init_ptrs_with_offsets(dst_gpr(), jep.dst_offsets);
+
+        uni_li(reg_work_amount, static_cast<int>(jep.work_amount));
+    }
+
+    // TODO: Support any LMUL values: get_max_lmul(exec_prc)
+    //       Currenly only aux_vec registers don't support any LMUL values (only m1):
+    //       jit emitters should known exact value of LMUL to preserve vec regs with correct idxs
+    exec_lmul = LMUL::m1;
+    exec_sew = bytes2sew(exec_prc.size());
+    OPENVINO_ASSERT(lmul2float(exec_lmul) <= lmul2float(get_max_lmul(exec_prc)),
+                    "Unsupported LMUL configuration for the kernel");
+
+    // We set other values for `current` variables to force first call `update_vlen`
+    current_lmul = LMUL::m2;
+    current_sew = SEW::e8;
+
+    for (size_t i = 0; i < jep.inputs_number; i++) {
+        if (jep.src_size[i] == 1) {
+            load_vector(i, src_gpr(i), reg_work_amount, jep_.src_prc[i], exec_prc, true);
+        }
+    }
+
+    size_t min_src_size = jep.dst_size;
+    for (size_t i = 0; i < jep.inputs_number; i++) {
+        if (jep.src_size[i] != 1)
+            min_src_size = std::min(min_src_size, jep.src_size[i]);
+    }
+
+    if (min_src_size == jep.dst_size) {
+        Label loop_begin;
+        Label loop_end;
+
+        L(loop_begin);
+        {
+            beqz(reg_work_amount, loop_end);
+
+            // We should call `update_vlen` on each loop iteration to get correct `reg_vlen` in iterations:
+            // `rg_vlen` should be valid in in tail loop
+            update_vlen(reg_work_amount, exec_sew, exec_lmul, true);
```

Comment:
Same story as for SVE: lets check perf overhead for vlen recomputation on each iteration. Maybe it is more benefitial to have separate tail loop.